### ISSUE TITLE
Streamline Transaction Data

### ISSUE THREAD

**Issue by nmorduch** [littleweaver/django-argus#41]
For the transaction form, I think we're asking for more data than people really want to record. Memo and notes often become redundant, and neither of them is really necessary. Can we just remove the memo field, and keep the optional notes field?

**Comment by melinath** [comment]
I disagree... Memo would be something like "Groceries", whereas notes could be a list of exactly what groceries were bought.

**Comment by nmorduch** [comment]
I think that the fact that it's groceries is going to be apparent from where it was bought, or is going to be a category.

**Comment by melinath** [comment]
It still seems useful to me as the "title" of the transaction. If you're displaying a transaction, you have the amount, and you have a short, one-line description of what it was (for example, who the payment was to, or an invoice number, or something else). This is pretty standard across the board for financial data - every bank I've used has had it, for example. Splitwise has it. Quickbooks has it. This is not IMO sufficiently replaced by categories, due to a lack of detail.

The notes field is specific to us (and splitwise) and allows for even more data to be attached. Hypothetically we could use it to replace the memo field. But how do we handle that? Notes is multi-line; memo is a single line. I guess we could do git commit style, where the first line is taken as the memo line and the rest is taken as additional information.

**Comment by nmorduch** [comment]
I'm not sure that such a title is necessary. I think that a recipient + an amount is the title to me. Some people want to keep specific notes, and I think that a multi-line notes field is sufficient for that, and some people want to have as little to enter as possible, which I think the required memo field gets in the way of.

I'm also considering whether there could be some sort of auto-fill on the memo, and whether that would help, but either it would be a pain to set up, or it would be just duplicating existing information so what would be the point.

Maybe another question here is whether we are considering displaying the logs in something other than table form. If we ever want to display in a simple list (which would have the details collapsed under each entry, I imagine), what would be the one or two things we would display? I suppose it would be memo and amount. Or maybe recipient and amount. Do we want to display it in that sort of collapsed list on mobile maybe?  I could warm to the idea.

**Comment by melinath** [comment]
As a simple house example, it makes a difference whether I go to Fred Meyer's for a shower curtain or groceries or a lamp.

... Also, recipient is currently optional. I think I would be okay with removing memo if recipient were required.
